spec/support/acts_as_tenant.rb

```diff
@@ -9,10 +9,12 @@ def with_tenant(tenant, &proc)
     context "with tenant #{tenant.inspect}" do |*params|
       before(:each) do
         ActsAsTenant.current_tenant = send(tenant)
+        @old_host = Capybara.app_host
```

Comment:
It's a different layer of abstraction. By placing the app host proper to be set by this helper, the host parsing concern for multitenancy can completely break without you know.

That should be a separate helper for multitenancy. You can define it in this file, but it should not mix with the model heler.